src/sidebar/test/bootstrap.js

```diff
@@ -13,3 +13,9 @@ sinon.assert.expose(assert, { prefix: null });
 window.jQuery = window.$ = require('jquery');
 require('angular');
 require('angular-mocks');
+
+// Configure Enzyme for UI tests.
+require('preact/debug');
+const { configure } = require('enzyme');
+const { Adapter } = require('enzyme-adapter-preact-pure');
+configure({ adapter: new Adapter() });
```

Comment:
Just digesting what I just googled...`enzyme` out of the box is for full-fat React, but it works with a couple of third-party "adapters", including preact

Reply:
You have to configure an adapter regardless of which library you use, the React ones are part of the Enzyme project on GitHub though.